ansible/roles/rancher/tasks/rancher.yml

```diff
@@ -5,10 +5,9 @@
 
 - name: Install Rancher
   command: >
-    helm install rancher-stable/rancher
-      --name rancher
+    helm install rancher rancher-stable/rancher
       --namespace cattle-system
-      --version {{ rancher_version | default('2.3.5', true) }}
+      --version {{ rancher_version | default('2.3.6', true) }}
```

Comment:
Should this default be here or in a vars file? I know you didn't add this here originally but it seems a litte out of place.

Reply:
Nope, in the var file makes more sense :+1: